packages/dcos-integration-test/extra/resiliency/test_aws_cf.py

```diff
@@ -59,15 +78,18 @@ def get_running_agents(group_name):
     dcos_api_session.post('/mesos/machine/down', json=down_hosts).raise_for_status()
 
     # Wait for replacements
-    test_util.helpers.wait_for_len(partial(get_running_agents, 'SlaveServerGroup'), len(dcos_api_session.slaves), 600)
     test_util.helpers.wait_for_len(
-        partial(get_running_agents, 'PublicSlaveServerGroup'), len(dcos_api_session.public_slaves), 600)
+        partial(list, dcos_stack.public_agent_instances),
+        len(dcos_api_session.public_slaves),
+        600)
+    test_util.helpers.wait_for_len(
+        partial(list, dcos_stack.private_agent_instances),
+        len(dcos_api_session.slaves),
+        600)
 
     # Reset the dcos_api_session to have the replacement agents
-    dcos_api_session.slaves = sorted([agent.private_ip_address for agent in
-                                      get_running_agents('SlaveServerGroup')])
-    dcos_api_session.public_slaves = sorted([agent.private_ip_address for agent in
-                                             get_running_agents('PublicSlaveServerGroup')])
+    dcos_api_session.slaves = get_private_ips(dcos_stack.private_agent_instances)
```

Comment:
That's touching dcos_api_session and outside of the scope of this PR in my opinion. If you change this, then you need to change  a few dozen other things throughout the codebase. I figured I would clean that up in the course of doing https://mesosphere.atlassian.net/browse/DCOS-13629 but I would also be fine if you wanted to file a JIRA for resolving that independently (the string change on its own should be easy, just tedious to find everywhere its used)